### ISSUE TITLE
Create a Jinja2 helper for external links

### ISSUE THREAD

**Issue by jmartinm** [inspirehep/inspire-next#585]
For consistency across the site, would be good if all external links (going away from INSPIRE) were displayed with an external icon, like:

![screen shot 2015-11-30 at 1 56 26 pm](https://cloud.githubusercontent.com/assets/580727/11472074/48a3cbbe-976a-11e5-9b54-a87960c9b3ee.png)

Ideally the developer would not have to worry about it and if a link points outside of the `inspirehep.net` domain the icon would be automatically added (with a possibility to disable that).

This could be done on the JavaScript side or on the server (Jinja2). I think the server side would be better as it would avoid elements appearing in the page after page load (once the JavaScript to do this trick runs).

So a good mechanism needs to be found to do this on Jinja2 (filter, macro or other)

**Comment by salmele** [comment]
TL;DR

One reason for which I like this is that a day we could work on the reference pages to make all reference clickable, to INSPIRE records when those exist, and outside with the external icon.

Full story:

Today in INSPIRE we have clickable titles (which is a hidden message for finding the record in INSPIRE) and non-clickable titles (which means we do not have the records in INSPIRE). This is a disservice to users which will need to cut and paste the non-clickable title in some search engine to try and find the article if so they wish. While we could have resolved it in a easier way by (1) acquiring that information and (2) make it clickable in a way that (3) does not confuse with an INSPIRE-only navigation.

For (1) however (a long-pending task of which @annetteholtkamp @tsgit know more) would be to try the CrossRef API or batch processing to extract as many DOIs as possible for the outgoing references not in INSPIRE. And other techniques/smarter use of metadata from publishers/and so on allow to acquire these DOIs as well.

Knowing this DOI will automatically fix (2) and your suggestion of icon elegantly address (3)

**Comment by nikpap** [comment]
Here's an idea of how to do it only with CSS: https://gist.github.com/01-Scripts/3010527
